src/js/components/common/forms/form.js

```diff
@@ -201,10 +201,13 @@ export default class Form extends React.Component {
   }
   _handleKeyPress(event) {
     event.stopPropagation();
-    if (event.key === 'Enter' && this.state.isValid) {
-      this.updateModel();
-    }
```

Comment:
side effect on the login screen, when focus is in the password entryfield (you just finished typing password) pressing 'Enter' does not submit the form -- you have to use the mouse.